src/commands/open/functions.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { cli } from 'cli-ux';
+import { Messages } from '@salesforce/core';
+import Command from '../../lib/base';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-functions', 'open.functions');
+
+export default class Open extends Command {
+  static summary = messages.getMessage('summary');
+
+  static description = messages.getMessage('description');
+
+  static examples = messages.getMessages('examples');
+
+  async run() {
+    const browserUrl = 'https://platform.salesforce.com/functions';
```

Comment:
- [ ] Still up in the air if this will be the final url where functions manager lives, needs confirmation.